extensions/applicationinsights-dependencies-js/src/ajaxRecord.ts

```diff
@@ -293,6 +293,12 @@ export class ajaxRecord {
                     method: self.method,
                     [strProperties]: { HttpMethod: self.method }
                 } as IDependencyTelemetry;
+
+                if (self.requestSentTime) {
```

Comment:
I suppose this is the actual fix, how was startTime being set before?

Reply:
Yes it is. The previous issue was that it wasn't getting set until the trackDependency call occurred, I've kept that code because it can still be called by external users and not set.

https://github.com/microsoft/ApplicationInsights-JS/blob/master/extensions/applicationinsights-dependencies-js/src/ajax.ts#L352-L354